sql/src/test/java/io/crate/analyze/EvaluatingNormalizerTest.java

```diff
@@ -123,17 +130,25 @@ public void testEvaluationClusterGranularity() {
         assertThat(query, instanceOf(Function.class));
     }
 
-    private FunctionInfo functionInfo(String name, DataType dataType, boolean isPredicate) {
-        ImmutableList<DataType> dataTypes;
+    private FunctionInfo functionInfo(String name, DataType<?> dataType, boolean isPredicate) {
```

Comment:
I had to adjust these tests for the operators as well and I went with an simpler approach. Just use this for functions with signatures:

```
    private FunctionInfo functionInfo(Signature signature, DataType dataType) {
        return functions.getQualified(signature, List.of(dataType, dataType)).info();
    }
```